### PR TITLE
chore: update ownership info for repo

### PR DESCRIPTION
**Pull request by jristau1984**
Continuing maintainership updates for 2U 2024.

### CODE REVIEW

**Comment by jristau1984** [comment]
@navinkarkera pinging you on this PR, as there appeared to be several other locations that claim some form of ownership on the repo.

**Comment by jristau1984** [comment]
I tagged you instead of replacing our team with your name because I had that exact same question.  I don't know what each and every one of these files is really supposed to do, so I think @feanil could offer better guidance.

File: openedx.yaml [hunk 1]
```diff
@@ -5,5 +5,5 @@ nick: lath
 oeps: {}
 owner:
   type: team
-  owner: edx/teaching-and-learning
+  owner:
```
Comment (connorhaugh) [inline openedx.yaml:8]:
```suggestion
  owner: ~
```

**Review by connorhaugh — CHANGES_REQUESTED** [review]
although it is ok that owner is not specified, per:

https://open-edx-proposals.readthedocs.io/en/latest/archived/oep-0002-bp-repo-metadata.html#id4

An empty space is not valid YAML, and this will fail reporting tools. Instead, please add '~' or 'null' or ''''.

**Review by connorhaugh — APPROVED** [review]

**Pull request merged**